### ISSUE TITLE
Earth map resource gone, globe doesn't render

### ISSUE THREAD

**Issue by kerel-fs** [poliastro/cesium-app#4]
The examples are using the following resource for the Earth map:
https://upload.wikimedia.org/wikipedia/commons/c/c4/Earthmap1000x500compac.jpg

As this resource is gone, the earth will not be rendered and the following error message appears in the console:
```
An error occurred in "SingleTileImageryProvider": Failed to load image https://upload.wikimedia.org/wikipedia/commons/c/c4/Earthmap1000x500compac.jpg.
```

**Comment by astrojuanlu** [comment]
Thanks for opening the issue @kerel-fs, I suspect it has to do with the new Cesium release, see:

* https://github.com/AnalyticalGraphicsInc/cesium/issues/8433
* https://github.com/poliastro/czml3/pull/52/ (our first attempt to upgrade)
* https://github.com/poliastro/czml3/pull/54/ (our final fix)

**Comment by kerel-fs** [comment]
Indeed, when opening the example files in Cesium Ion an Earth is shown.

So I guess there are two problems:
- the Earthmap from wikipedia referenced in the example files is gone
- cesium-app doesn't render a default Earth currently (in contrast to the latest Cesium in Cesium Ion or the czml3 Widget), probably related to the issues you linked.

**Comment by Sedictious** [comment]
Thanka for bringing this up @kerel-fs 

The cesium-app uses a texture by default in order to be able to render planetary bodies other than earth.

There are two potential fixes:

- Upload a texture map to the github repo as to not worry of a texture being potentially taken down

- Render the default Earth and have texture maps be optional

**Comment by astrojuanlu** [comment]
From the deletion logs:

https://commons.wikimedia.org/wiki/Commons:Deletion_requests/File:Earthmap1000x500compac.jpg

> Permission link doesn't contain any permission to publish this as PD.

For this app I would vote for @Sedictious option (2): Render the default Earth and have texture maps be optional.

**Comment by astrojuanlu** [comment]
More sources of textures that are free to use:

* NASA original Blue Marble https://earthobservatory.nasa.gov/features/BlueMarble/BlueMarble_2002.php
* Blue Marble Next Generation https://visibleearth.nasa.gov/collection/1484/blue-marble (one image per month)
* Textures for planets of the Solar System! https://www.solarsystemscope.com/textures/ (CC-BY 4.0)

**Comment by kerel-fs** [comment]
Another source for various moon and mars basemaps:
- https://www.openplanetary.org/opm/basemaps (free to use, but exact license unknown)
  (static rendered images: https://github.com/openplanetary/opm-basemaps/tree/master/static-map-images)

**Comment by Sedictious** [comment]
Is anyone aware of the license of NASA's image sets? I couldn't find anything but I guess we can't re-upload them and reference them from our repo...

**Comment by astrojuanlu** [comment]
@Sedictious this is the image use policy: https://visibleearth.nasa.gov/image-use-policy

Therefore, we should be allowed to reference or upload them in return of giving credit to NASA!
